app/src/main/res/layout/fragment_potential_risk.xml

```diff
@@ -0,0 +1,285 @@
+
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:drawableStart="@drawable/check"
+                    android:drawablePadding="24dp"
+                    android:paddingStart="24dp"
+                    android:text="@string/isolate_from_others"
+                    android:textSize="18sp"
+                    tools:ignore="RtlSymmetry" />
+
+            </LinearLayout>
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="24dp"
+                android:layout_marginBottom="16dp"
+                android:layout_weight="1"
+                android:orientation="horizontal"
+                android:layout_marginHorizontal="32dp">
+
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:drawableStart="@drawable/check"
+                    android:drawablePadding="24dp"
+                    android:text="@string/rest_and_take_care"
+                    android:textSize="18sp"
+                    android:paddingStart="24dp"
+                    tools:ignore="RtlSymmetry" />
+
+            </LinearLayout>
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="24dp"
+                android:layout_marginBottom="16dp"
+                android:layout_weight="1"
+                android:orientation="horizontal"
+                android:layout_marginHorizontal="32dp">
+
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:drawableStart="@drawable/check"
+                    android:drawablePadding="24dp"
+                    android:paddingStart="24dp"
+                    android:text="@string/get_tested_if_advised"
+                    android:textSize="18sp"
+                    tools:ignore="RtlSymmetry" />
+
+            </LinearLayout>
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:orientation="horizontal"
+                android:layout_marginHorizontal="32dp"
+                android:layout_marginBottom="32dp">
+
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:drawableStart="@drawable/check"
+                    android:drawablePadding="24dp"
+                    android:paddingStart="24dp"
+                    android:text="@string/refer_to_local_agencies"
+                    android:textSize="18sp"
+                    tools:ignore="RtlSymmetry" />
+
+            </LinearLayout>
```

Comment:
With `drawableStart` set we don't need this `LinearLayout`s anymore

Reply:
We have to make our layout as flat as possible:
`<LinearLayout orientation="horizontal"> <ImageView/> <TextView/> <LinearLayout/>` is the same as writing `<TextView drawableStart=".."/>`

Reply:
@madim done.